parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -402,6 +398,18 @@ cleanUpContainerd() {
     rm -Rf $CONTAINERD_DOWNLOADS_DIR
 }
 
+configureGPUDrivers() {
+  if $FULL_INSTALL_REQUIRED; then
+        installGPUDrivers
+    fi
```

Comment:
This "fi" is not aligned with its corresponding "if" above.

Reply:
Done